psique.gemspec

```diff
@@ -1,30 +1,56 @@
+  s.test_files = ["test/test_helper.rb", "test/psique_test.rb", "test/integration", "test/integration/navigation_test.rb", "test/dummy", "test/dummy/public", "test/dummy/public/422.html", "test/dummy/public/404.html", "test/dummy/public/favicon.ico", "test/dummy/public/500.html", "test/dummy/app", "test/dummy/app/models", "test/dummy/app/models/concerns", "test/dummy/app/assets", "test/dummy/app/assets/images", "test/dummy/app/assets/images/logo-mobile.jpg", "test/dummy/app/assets/images/mockup.png", "test/dummy/app/assets/javascripts", "test/dummy/app/assets/javascripts/application.js", "test/dummy/app/assets/javascripts/test.js", "test/dummy/app/assets/stylesheets", "test/dummy/app/assets/stylesheets/application.css", "test/dummy/app/views", "test/dummy/app/views/test", "test/dummy/app/views/test/index.html.haml", "test/dummy/app/views/layouts", "test/dummy/app/views/layouts/application.html.haml", "test/dummy/app/controllers", "test/dummy/app/controllers/application_controller.rb", "test/dummy/app/controllers/test_controller.rb", "test/dummy/app/controllers/concerns", "test/dummy/app/helpers", "test/dummy/app/helpers/test_helper.rb", "test/dummy/app/helpers/application_helper.rb", "test/dummy/app/mailers", "test/dummy/test", "test/dummy/test/controllers", "test/dummy/test/controllers/test_controller_test.rb", "test/dummy/lib", "test/dummy/lib/assets", "test/dummy/Rakefile", "test/dummy/config", "test/dummy/config/environment.rb", "test/dummy/config/locales", "test/dummy/config/locales/en.yml", "test/dummy/config/environments", "test/dummy/config/environments/production.rb", "test/dummy/config/environments/development.rb", "test/dummy/config/environments/test.rb", "test/dummy/config/routes.rb", "test/dummy/config/initializers", "test/dummy/config/initializers/mime_types.rb", "test/dummy/config/initializers/backtrace_silencers.rb", "test/dummy/config/initializers/session_store.rb", "test/dummy/config/initializers/wrap_parameters.rb", "test/dummy/config/initializers/assets.rb", "test/dummy/config/initializers/cookies_serializer.rb", "test/dummy/config/initializers/inflections.rb", "test/dummy/config/initializers/filter_parameter_logging.rb", "test/dummy/config/application.rb", "test/dummy/config/secrets.yml", "test/dummy/config/boot.rb", "test/dummy/config/database.yml", "test/dummy/config.ru", "test/dummy/README.md", "test/dummy/bin", "test/dummy/bin/setup", "test/dummy/bin/bundle", "test/dummy/bin/rails", "test/dummy/bin/rake"]
 
-  s.add_dependency 'rails'
-  s.add_dependency 'sass-rails'
-  s.add_dependency 'coffee-rails'
-  s.add_dependency 'normalize-rails'
-  s.add_dependency 'font-awesome-rails'
-  s.add_dependency 'haml'
-  s.add_dependency 'bourbon'
-  s.add_dependency 'neat'
+  if s.respond_to? :specification_version then
+    s.specification_version = 4
 
-  s.add_development_dependency 'sqlite3'
+    if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
```

Comment:
:scream: What's going on there?
Not sure if it's worth.. afaik `add_dependency` is supported on newer versions of gem. Is it going to be deprecated?
Any reason for that ugly change?